src/components/UI/Text/InputNumber.vue

```diff
@@ -257,7 +262,14 @@ export default {
     },
     keyPress(event) {
       // invoked when a key is pressed
-      if (this.maxLength != null && this.localValue.length == this.maxLength) {
+      // character key not accepted
+      event = event || window.event;
```

Comment:
What is the purpose of `window.event`?

Reply:
It returns the Event which is currently being handled by the site's code, not required here as we are already specifying the event.